package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {},
   "dependencies": {
     "acorn": "^8.15.0",
-    "node-repl-polyfill": "github:CanadaHonk/node-repl-polyfill"
+    "node-repl-polyfill": "^0.1.2"
```

Comment:
@CanadaHonk Let me know if you prefer to keep the latest behavior. We could change this to `latest` if you prefer, but I think its best to utilize semver to avoid accidentally breaking porf.